kitsune/customercare/zendesk.py

```diff
@@ -7,14 +7,23 @@
 
 # See docs/zendesk.md for details about getting the valid choice values for each field:
 CATEGORY_CHOICES = [
-    (None, _lazy("Select a topic")),
+    (None, _lazy("Select a reason for contacting")),
     ("payment", _lazy("Payments & Billing")),
     ("accounts", _lazy("Accounts & Login")),
     ("technical", _lazy("Technical")),
     ("feedback", _lazy("Provide Feedback/Request Features")),
     ("not_listed", _lazy("Not listed")),
 ]
 
+CATEGORY_CHOICES_LOGINLESS = [
+    (None, _lazy("Select a reason for contacting")),
+    ("fxa-reset-password", _lazy("I forgot my password")),
```

Comment:
If these are placeholders, let's add an issue to update them when we have the final values

Reply:
Joe sent me what will be the "real" values.  Adding and updating.

Reply:
Learned they all post to a different field, too - so changes incoming for that.